ipq-wifi: update 8devices Habanero/Jalapeno BDFs

mantas-p opened the pull request:
Updatd BDFs fixes temperature sensor readout.

Boardfiles have been submitted upstream:
http://lists.infradead.org/pipermail/ath10k/2023-March/014297.html
http://lists.infradead.org/pipermail/ath10k/2023-March/014298.html

Also update MeshPoint.One as it uses Jalapeno SoM

BKPepe commented on target/linux/ipq40xx/image/generic.mk:
```diff
@@ -336,7 +338,7 @@ define Device/cilab_meshpoint-one
 	DEVICE_VENDOR := Crisis Innovation Lab
 	DEVICE_MODEL := MeshPoint.One
-	DEVICE_PACKAGES := kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
+	DEVICE_PACKAGES += kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
```
I think this should be done in separate commit as it might seems that you are doing something else than what is said in the commit subject and commit description

mantas-p replied:
You are right, I failed to explicitly mention MeshPoint.One thinking it as subclass of Jalapeno. Updated now.

mantas-p committed:
ipq40xx: update Crisis Innovation Lab MeshPoint.One BDF

Make sure it uses updated Jalapeno BDF inherited from
Device/8dev_jalapeno-common

Fixes: 146eb4925c7f ("ipq40xx: add support for Crisis Innovation Lab MeshPoint.One")
Signed-off-by: Mantas Pucka <mantas@8devices.com>
[ fix Fixes tag to correct format and fix commit title ]
Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>

Ansuel commented:
@mantas-p BDF got moved in a separate repo can you resubmit there? https://github.com/openwrt/firmware_qca-wireless

(ignore that it's now empty... will be updated at the first chance when i update the git repo)

mantas-p commented:
@Ansuel I don't think it's possible to make a PR to an empty repo. Or do you want me to send BDFs through the mailing list?

BKPepe commented:
It should be possible, but still, this can go to the repository as it is, and it can be moved anytime there and it will be done as treewide commit, which is probably what we want. Let's not add more overhead to contributors.

Ansuel commented:
Main problem is the makefile change so that is why I can't take this as is.

Il Gio 16 Mar 2023, 10:09 Josef Schlehofer ***@***.***> ha
scritto:

> It should be possible, but still, this can go to the repository as it is,
> and it can be moved anytime there and it will be done as treewide commit,
> which is probably what we want. Let's not add more overhead to contributors.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openwrt/openwrt/pull/12139#issuecomment-1471567825>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE2ZMQRNKIQEPZWPVIV4MKTW4LKF7ANCNFSM6AAAAAAVTYPFAI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mantas-p commented:
@Ansuel I've rebased this, but I can't make a PR to https://github.com/openwrt/firmware_qca-wireless because it's empty. Can you just take changes from: https://github.com/mantas-p/qca-wireless ?

Ansuel commented:
@mantas-p done!

Ansuel commented:
Now you should bump the ipq-wifi package version and you should be good to go.

mantas-p committed:
ipq-wifi: bump to latest git HEAD and add 8devices boards

2dae618 ipq-wifi: update 8devices Jalapeno BDF
08e92db ipq-wifi: update 8devices Habanero BDF

Signed-off-by: Mantas Pucka <mantas@8devices.com>
[ split ipq40xx changes in separate commit ]
Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>

mantas-p committed:
ipq40xx: set specific BDF file for 8devices Habanero/Jalapeno

Set specific BDF file for 8devices Habanero/Jalapeno in ipq40xx
generic.mk

Signed-off-by: Mantas Pucka <mantas@8devices.com>
[ split ipq40xx changes in separate commit ]
Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>

Ansuel commented:
@mantas-p i found time to fix this and I split 2 commits. 

If the changes I did are ok for you I will merge this. (asking since i split the commit and still used you as author so asking permission for the change)

mantas-p commented:
@Ansuel thanks, looks good, please merge.
